spec/services/meeting_starter_spec.rb

```diff
@@ -26,6 +26,10 @@
 
   describe '#call' do
     it 'calls BigBlueButtonApi with the right params' do
+      allow_any_instance_of(BigBlueButtonApi)
+        .to receive(:start_meeting)
+        .and_return(meeting_starter_response)
+
```

Comment:
This is fine for this PR but we may want to move this to a `before` similar to `MeetingsControllerSpec`

Reply:
Right. Might as well.